osu.Game/Rulesets/Difficulty/DifficultyCalculator.cs

```diff
@@ -99,7 +102,7 @@ public IEnumerable<DifficultyAttributes> CalculateAll()
                 if (combination is MultiMod multi)
                     yield return Calculate(multi.Mods);
                 else
-                    yield return Calculate(combination);
+                    yield return Calculate(new[] { combination });
```

Comment:
nitpick but there is an extension for this
```suggestion
                    yield return Calculate(combination.Yield());
```